### PR TITLE
Service user test1

### PR DESCRIPTION
**Pull request by yuhai**
Move integration tests of com.liferay.portal.service.user package to module. Please check this after ci:test finished.

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:help

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/service/user/test/UserServiceWhenCompanySecurityStrangersWithMXDisabledTest.java [hunk 1]
```diff
@@ -18,6 +18,9 @@
 import com.liferay.portal.kernel.exception.UserEmailAddressException;
 import com.liferay.portal.kernel.model.User;
 import com.liferay.portal.kernel.security.auth.PrincipalThreadLocal;
+import com.liferay.portal.kernel.security.permission.PermissionChecker;
+import com.liferay.portal.kernel.security.permission.PermissionCheckerFactoryUtil;
```
Comment (dantewang) [inline modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/service/user/test/UserServiceWhenCompanySecurityStrangersWithMXDisabledTest.java]: @yuhai Please inject `PermissionCheckerFactory`

**Comment by dantewang** [comment]
Pull request submitted to https://github.com/yuhai/liferay-portal/pull/131. [See changes here.](https://github.com/dantewang/liferay-portal/compare/151b698d2cf0~6...151b698d2cf0) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/service/user/test/UserServiceWhenPortalSendsPasswordEmailTest.java [hunk 2]
```diff
@@ -250,6 +288,37 @@ protected PortletPreferences givenThatCompanySendsResetPasswordLink()
 		return portletPreferences;
 	}
 
+	protected void replaceClassLoader(Localization localization) {
```
Comment (dantewang) [inline modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/service/user/test/UserServiceWhenPortalSendsPasswordEmailTest.java]: @yuhai I SFed this class and sent you a pull: https://github.com/yuhai/liferay-portal/pull/131

**Comment by dantewang** [comment]
@yuhai I didn't check every moved class, please also take a look at them.

**Pull request closed without merging**